SDPi_Supplement/asciidoc/volume1/use-cases/tf1-ch-c-use-case-stad.adoc

```diff
@@ -0,0 +1,65 @@
+// = Use Case Feature: Synchronized Time Across Devices (STAD)
+
+#TODO:  Add use case metadata strategy#
+
+// [mdi_use_case#use_case_stad,actors='actor_somds_provider actor_somds_consumer',figure=vol2_figure_dev_24_probe_sequence,messages='message_announce_network_presence']
+
+// FROM TF-2 DEV-24 Transaction document
+// [sdpi_transaction#transaction_dev_24,actors='actor_somds_provider actor_somds_consumer',figure=vol2_figure_dev_24_probe_sequence,messages='message_announce_network_presence']
+
+[sdpi_offset=2]
+=== Use Case Feature {var_use_case_id}: Synchronized Time Across Devices (STAD)
+
+==== Narrative:
+Nurse Jean attaches a ventilator to the SDC based network in the ICU.  It automatically synchronizes its clock with the clocks of the other devices on the network.
+
+==== Technical View
+
+.Synchronized Time Across Devices (STAD) -- Technical View
+
+image::../../images/vol1-diagram-use-case-stad-tech-view.svg[align=center]
+
+==== Technical Pre-Conditions
+
+*Given* All devices communicate using SDC
+
+*And* NTP service on the SDC network
+
+==== Scenarios
+
+===== Scenario: STAD {var_use_case_id}.1 - Device is connected to network with NTP service
+
+*Given* Device has detected at least one NTP service
+
+*When* The NTP service is operational
+
+*Then* The device will synchronize its time with the NTP service time
+
+===== Scenario: STAD {var_use_case_id}.2 - Device is connected to network with a failed NTP server
+
+*Given* Device has not detected at least one NTP service
+
+*When* The NTP service is not operational or inaccessible
+
+*Then* The device will synchronize its time with another SDC device in the cluster
```

Comment:
- Is the distinction of "device" and "SDC device" on purpose?
- What are the boundaries of a "cluster"
- How do we synchronize time with other SDC devices? From what I know, devices can synchronize with NTP servers by using NTP server addresses. Those addresses are either configured by service personell or retrieved from the MDIB of other SDC devices. A sync with another device is currently out-of-scope for SDC. Would needed to be profiled.

Reply:
Wording updated.